pkg/api-server/inspect_endpoints.go

```diff
@@ -111,7 +111,10 @@ func inspectPolicies(
 			return
 		}
 
-		result := &api_server_types.PolicyInspectEntryList{}
+		result := &api_server_types.PolicyInspectEntryList{
```

Comment:
Sometimes it's "use `nil`" and sometimes its "use `[]{}`" for the empty slice. This seems like a really silly issue that will always come back. Can we enforce this via constructors in the `api_server_types` or custom marshallers or similar? Of course [proposals to improve this](https://github.com/golang/go/issues/37711) in the stdlib are blocked because getting one of go's many leaky abstractions recognized is like pulling teeth.

Reply:
Thanks, I added a couple of constructors